recipes/pymultinest/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: pymultinest
+  version: "2.6"
+
+source:
+  fn: pymultinest-2.6.tar.gz
+  url: https://pypi.python.org/packages/c2/fc/77bdd8356068125bba0a5c07bfa2b4a3f492bff614aeae783bc1a566d699/pymultinest-2.6.tar.gz
+  md5: d9e5eee6abd3784056ca5ccc1f05cab9
+
+build:
+   number: 1
+   skip: true  # [win]
+   skip: true  # [py3k]
+
+requirements:
+
+  build:
+    - toolchain
+    - python x.x
```

Comment:
Please see the new recommendations for building against numpy:

https://conda-forge.org/docs/meta.html#building-against-numpy

Reply:
ok, I followed those, thanks for the pointer.